dev/com.ibm.websphere.appserver.jaxws-2.2/com.ibm.websphere.appserver.jaxws-2.2.feature

```diff
@@ -63,4 +63,4 @@ Subsystem-Name: Java Web Services 2.2
  bin/jaxws/tools/ws-wsgen.jar
 kind=ga
 edition=base
-IBM-Maven-Dependency: javax.xml.ws:jaxws-api:2.3
\ No newline at end of file
+IBM-Maven-Dependency: javax.xml.ws:jaxws-api:2.2
```

Comment:
Base on https://repo.maven.apache.org/maven2/javax/javaee-api/7.0/javaee-api-7.0.pom, we should use javax.xml.ws:jaxws-api:2.2.8